app/views/analytics/districts/show.csv.erb

```diff
@@ -65,6 +65,7 @@ end).to_csv -%>
 <%
   headers = [
     "Facility name",
+    "Facility type",
```

Comment:
nitpick of the day: `Facility Type`?

Reply:
In fact, changed the other one to `Facility type` for consistency instead.